samples/nrf_rpc/ps_client/sample.yaml

```diff
@@ -0,0 +1,37 @@
+sample:
```

Comment:
Is any loopback required to run this sample? If yes, please add information about the obligatory use of the fixture.

Reply:
Hi @katgiadla, documentation regarding the sample is in separate PR: https://github.com/nrfconnect/sdk-nrf/pull/16597
it is blocked until this is merged. Please take a look there if you can find there anything regarding your question.

Reply:
it's not needed.